app/full_text_search.py

```diff
@@ -131,6 +131,22 @@ def create_jp_name_fields(**kwargs):
                         value=romanized_japanese_name)
                 )
                 romanized_names_list.append(romanized_japanese_name)
+                kwargs[field] = romanized_japanese_name
```

Comment:
In general, you should avoid modifying method parameters. Can you define another variable to store this?